packages/trade-widget/src/types.ts

```diff
@@ -0,0 +1,65 @@
+import { Order, SignedOrder } from '0x.js';
+import { BigNumber } from '@0xproject/utils';
+import * as Web3 from 'web3';
+
+export type ArtifactContractName = 'Forwarder';
```

Comment:
This should be in a `constants.ts` or `configs.ts` file as `ARTIFACT_CONTRACT_NAME`.